src/services/userServices.js

```diff
@@ -1,34 +1,31 @@
 const userModel = require('../models/userModel')
 // const bcrypt = require('bcrypt')
 
-const getUsers = async (req, res) => {
+const getUsers = async () => {
     try {
         const users = await userModel.find()
         if (!users) {
             return "Users not found"
         }
         return users
     } catch (err) {
-        return Promise.reject({ status: 404, message: 'Resource Not Found' })
+        return Promise.reject('Cannot find users -> ' + err)
     }
 }
 
-const getUser = async (req) => {
-    const { id } = req.params
-
+const getUser = async (id) => {
     try {
         const user = await userModel.findById(id)
         if (!user) {
             return "User not found"
         }
         return user
     } catch (err) {
-        return Promise.reject({ status: 404, message: 'Resource Not Found' })
+        return Promise.reject('Cannot find user -> ' + err)
     }
 }
 
-const addUser = async (req) => {
-    const { userName, password, email, phoneNumber, role } = req.body
+const addUser = async (userName, password, email, phoneNumber, role) => {
```

Comment:
Here you can send an object that contains all of this info and after that to destruvture it